pkg/providers/aws/provider_postgres.go

```diff
@@ -210,6 +210,7 @@ func (p *PostgresProvider) CreatePostgres(ctx context.Context, pg *v1alpha1.Post
 		}
 		logger.Infof("created security group %s", aws.StringValue(securityGroup.StandaloneSecurityGroup.GroupName))
 	}
+	return nil, "don't create the postgres", errorUtil.New("dont create the postgres")
```

Comment:
This will be removed, it's just to allow testing of the network creation without creating a postgres.

Reply:
I was expecting a less serious error message :rofl:

Reply:
How about if I add a booooop? 😝

Reply:
That would be too many o`s, would have to close the MR